tmt/hardware.py

```diff
@@ -507,7 +507,8 @@ def _from_specification(
             as_quantity: bool = True,
             as_cast: Optional[Callable[[str], ConstraintValueT]] = None,
             original_constraint: Optional['Constraint[Any]'] = None,
-            allowed_operators: Optional[list[Operator]] = None
+            allowed_operators: Optional[list[Operator]] = None,
+            default_unit: Optional[Any] = "bytes"
```

Comment:
Just a minor suggestion. Could you also add a docstring for the `default_unit`, as every other parameter has one as well?